plugins/com.google.cloud.tools.eclipse.appengine.facets.test/src/com/google/cloud/tools/eclipse/appengine/facets/StandardFacetInstallDelegateTest.java

```diff
@@ -56,32 +56,36 @@ public void testCreateConfigFiles()
     delegate.createConfigFiles(project, AppEngineStandardFacet.JRE7, monitor);
 
     IFile appengineWebXml = project.getFile("src/main/webapp/WEB-INF/appengine-web.xml");
+    IFile loggingProperties = project.getFile("src/main/webapp/WEB-INF/logging.properties");
     Assert.assertTrue(appengineWebXml.exists());
-    
+    Assert.assertTrue(loggingProperties.exists());
+
     try (InputStream in = appengineWebXml.getContents(true)) {
       AppEngineDescriptor descriptor = AppEngineDescriptor.parse(in);
       assertNull(descriptor.getRuntime());
     }
   }
   
   @Test
-  public void testCreateConfigFiles_dontOverwrite() 
-      throws CoreException, IOException {
-    
+  public void testCreateConfigFiles_dontOverwrite() throws CoreException, IOException {
     IFolder webInfDir = project.getFolder("src/main/webapp/WEB-INF");
     ResourceUtils.createFolders(webInfDir, monitor);
-    IFile appengineWebXml = project.getFile("src/main/webapp/WEB-INF/appengine-web.xml");
+    IFile appengineWebXml = webInfDir.getFile("appengine-web.xml");
+    IFile loggingProperties = project.getFile("logging.properties");
```

Comment:
This feels like two methods are getting mixed together. One should be able to fail independently of the other.